apps/api/src/app/config/sample.env

```diff
@@ -8,5 +8,5 @@ ARANGO_DB_USER_PASSWORD=youllNEVERguess1!
 ARANGO_DB_NAME=sampledb
 AUTH0_ISSUER_URL=https://foo.auth0.com/
 AUTH0_AUDIENCE=https://bar.mydomain.ca
-BASE_DIGITAL_ASSET_URL=https://www.myaudio.com:5555/media?id=
```

Comment:
We don't want any real data in the `sample.env`. Reset this change.

Reply:
I thought I was editing development.env at the time and realized my mistake.  But then I must have forgotten to go back and fix sample.env